### ISSUE TITLE
Welcome for any question and propose new features

### ISSUE THREAD

**Issue by NLPJCL** [NLPJCL/RAG-Retrieval#1]
thank you

**Comment by cviviers** [comment]
Hi @NLPJCL, 
Great work! Do you have code available for the vision language finetuning? I am considering finetuning your model on a new domain.

**Comment by baochi0212** [comment]
Hi @NLPJCL, I sanity check with bge-en-icl embedding distill to bert model with ArguAna datasets following guides:
- create mmap from https://github.com/NovaSearch-Team/RAG-Retrieval/tree/master/examples/stella_embedding_distill
- train with "distill" mode but it seems like we lack accelerator input in DistillEmbedding forward
when I add accelerator to forward func, the loss encouters NaN bug, any idea on this, thanks.

**Comment by NLPJCL** [comment]
> Hello [@NLPJCL](https://github.com/NLPJCL) ,
> 
> Thanks for all the good ideas in this repository. Would you be able to give an end to end example on how to distill a LLM Reranker into a Bert base model. I'd like to distill Qwen3-Reranker into a XLM-R model using your code.
> 
> Thanks

Thank you for using our framework. After you predict the socre, you can directly use the pointe-wise method to learn.
We will optimize the readme to make it clearer and easier to understand.
@Ziyang1060
